test/integration/host_js_test.rb

```diff
@@ -529,6 +529,7 @@ class HostJSTest < IntegrationTestWithJavascript
       page.find(id).click
       assert page.has_no_selector?(id)
       click_button('Submit')
+      find('h5', :text => host.fqdn) # wait for the host page to load
```

Comment:
There is a dedicated `click_on_submit` helper (introduced in 1580adbec7995b3d27e55d4067a2d5a4fd34d291), but all of its uses were removed in 98c17528a655148bf4c01629ea36cf249441c213 and I am not sure why.

Following the same logic as in 98c17528a655148bf4c01629ea36cf249441c213 and using a `find` to find the header with the hostname.

Reply:
Split that change out to https://github.com/theforeman/foreman/pull/10302